modules/bibauthorid/lib/bibauthorid_daemon.py

```diff
@@ -149,7 +171,26 @@ def _task_submit_elaborate_specific_parameter(key, value, opts, args):
         value = [token.strip() for token in value.split(',')]
         bibtask.task_set_option("last-names", value)
     elif key in ("--single-threaded",):
-        bibtask.task_set_option("single-threaded", True)
+        bibtask.task_set_option("single_threaded", True)
+    elif key in ("--force-identifier-consistency",):
+        bibtask.task_set_option("force_identifier_consistency", True)
+    elif key in ("--check-db",):
+        bibtask.task_set_option("check_db", True)
+    elif key in ("--do-not-modify-records",):
+        bibtask.task_set_option("dry_run", True)
+    elif key in ("--packet-size"):
+        if value.count("="):
+            value = value[1:]
+        value = int(value)
+        bibtask.task_set_option("packet_size", value)
+    elif key in ("--do-not-modify-hepnames",):
+        bibtask.task_set_option("dry_hepnames_run", True)
+    elif key in ("--open-tickets",):
+        bibtask.task_set_option("open_tickets", True)
+    elif key in ("--queue",):
+        if value.count("="):
+            value = value[1:]
+        bibtask.task_set_option("queue", value)
```

Comment:
ah I see... if above you put `=`, then in value you will indeed have the very correct value